web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
That is probably the case, but it's not sufficient to make it the homepage.

We need to consider:
  * Initial new user reactions
  * Support costs for us as developers
  * Discoverability of the various statusy information for users
  * Non-new user use cases

If the argument is that /graph is more used than /status, then start page should be the console templates as that's even more popular.

I think we should be looking to improve the current status page, as /graph is overall not a better experience for us and our users.